treebeard/tests/test_treebeard.py

```diff
@@ -2355,24 +2355,14 @@ def test_treebeard_js(self):
         rendered = template.render(context)
         expected = ('<script type="text/javascript" src="jsi18n"></script>'
                     '<script type="text/javascript" '
-                    'src="/treebeard/treebeard-admin.js"></script>'
-                    '<script>(function($){'
+                    'src="{adminjs}"></script>'
+                    '<script>(function($){{'
                     'jQuery = $.noConflict(true);'
-                    '})(django.jQuery);</script>'
+                    '}})(django.jQuery);</script>'
                     '<script type="text/javascript" '
-                    'src="/treebeard/jquery-ui-1.8.5.custom.min.js"></script>')
+                    'src="{jqy}"></script>').format(adminjs=static('treebeard/treebeard-admin.js'), jqy=static('treebeard/jquery-ui-1.8.5.custom.min.js'))
```

Comment:
Do to curly braces in the JS code, I think `.format` is maybe not the best option to string interpolation here. How about

- a) `'..' % (..)`
- b) `".." + ... + ".."`

instead?

Reply:
Updated along with commit clean ups